golang/processor.go

```diff
@@ -1,6 +1,7 @@
 package main
 
 import (
+	kavka "example/hello/pkg/kafka"
```

Comment:
Kafka is a [Czech](https://en.wikipedia.org/wiki/Czech_language) surname, which is an old spelling of the word "kavka"